htsinfer/rand_nuc_seq.py

```diff
@@ -0,0 +1,249 @@
+from collections import Counter
```

Comment:
Please place the code/function in the `htsinfer/infer_read_layout.py` module rather than in its own module